lib/probe/definitions.ex

```diff
@@ -109,8 +109,9 @@ defmodule Instruments.Probe.Definitions do
   defp unique_names(probe_names, options) do
     case Keyword.get(options, :tags) do
       tags when is_list(tags) ->
-        for probe_name <- probe_names, tag <- tags do
-          "#{probe_name}.tag:#{tag}"
+        tag_string = inspect(Enum.sort(tags))
```

Comment:
nit, perhaps `Enum.join` this instead of inspecting it?

Reply:
I second this nit.  The call to inspect here can potentially lead to two unique lists of tags being incorrectly considered the same list.

By default `Kernel.inspect/2` sets the `:limit` option to 50.  There is a small edge case here where someone defines more than 50 tags (I fully admit this is a crazy number of tags) and the unique tags end up at the end of the list after sorting.  `Kernel.inspect/2` will elide any element after the 50th element with `", ... ]"`

Reply:
Thanks, that makes good sense. This is a public library so it's definitely important to get it strictly correct.